packages/libs/react-ui/src/components/Select/Select.test.tsx

```diff
@@ -0,0 +1,89 @@
+import { Select } from '@components/Select';
+import { Option } from '@components/Select/Option';
+import { fireEvent, render } from '@testing-library/react';
+import React from 'react';
+
+describe('Select', () => {
+  it('renders without errors', () => {
+    const { getByTestId } = render(
+      <Select value="1" onChange={() => {}}>
+        <Option value="1">Option 1</Option>
+        <Option value="2">Option 2</Option>
+      </Select>,
+    );
+
+    const selectContainer = getByTestId('kda-select');
+    expect(selectContainer).toBeInTheDocument();
```

Comment:
When doing the exact same thing in the next test, we don't need this one? We probably do this more often and I don't see any benefit tbh. One test doesn't matter much but it adds up.

Reply:
this one just tests container and rest mostly focusing on select, options and functionality, but yes indeed this one doesn't add much

Reply:
Everything that happens in this test happens in the others. So why keep it?